test/src/test/java/hudson/cli/RunRangeCommand2Test.java

```diff
@@ -84,7 +84,7 @@ public class RunRangeCommand2Test {
 
     @Test public void dummyRangeShouldSuccessEvenTheBuildIsRunning() throws Exception {
         FreeStyleProject project = j.createFreeStyleProject("aProject");
-        project.getBuildersList().add(Functions.isWindows() ? new BatchFile("echo 1\r\nping -n 10 127.0.0.1 >nul") : new Shell("echo 1\nsleep 10s"));
```

Comment:
Would https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/SleepBuilder.html not be better suited for these tests? Lots of `#waitForMessage` wait for the output of that step.